extensions/2.0/Vendor/BENTLEY_primitive_restart/README.md

```diff
@@ -0,0 +1,152 @@
+            "bufferView": 0,
+            "byteOffset": 3,
+            "count": 3,
+            "componentType": 5121,
+            "type": "SCALAR",
+            "name": "Line string 2 indices"
+        },
+        {
+            "bufferView": 0,
+            "componentType": 5121,
+            "count": 6,
+            "type": "SCALAR",
+            "name": "All indices"
+        },
+        {
+            "bufferView": 1,
+            "componentType": 5126,
+            "count": 5,
+            "type": "VEC3",
+            "max": [
+                0.5,
+                0.5,
+                0.0
+            ],
+            "min": [
+                -0.5,
+                -0.5,
+                0.0
+            ],
+            "name": "Positions accessor"
+        }
+    ],
+
+```
+
+The mesh looks like this:
+
+```json
+    "meshes": [
+        {
+            "primitives": [
+                {
+                    "attributes": {
+                        "POSITION": 0
+                    },
+                    "indices": 0,
+                    "material": 0,
+                    "mode": 3
+                },
+                {
+                    "attributes": {
+                        "POSITION": 0
+                    },
+                    "indices": 1,
+                    "material": 0,
+                    "mode": 3
+                }
+            ],
+            "extensions": {
+                "BENTLEY_primitive_restart": {
+                    "primitiveGroups": [
+                        {
+                            "primitives": [
+                                0,
+                                1
+                            ],
+                            "indices": 2
+                        }
+                    ]
+                }
+            }
+        }
+    ],
+```
+
+By default, this mesh draws two separate line strip primitives, each using its own `indices` accessor. The `BENTLEY_primitive_restart` extension specifies that both primitives can be replaced with a single one using a combined `indices` accessor containing primitive restart values.
+
+## glTF Schema Updates
+
+The `BENTLEY_primitive_restart` extension is applied to a mesh. Its `primitiveGroups` property is a list of groups of primitives that can be replaced with a single primitive using primitive restart. Each group is described by a list of indices into `mesh.primitives`, along with the ID of the accessor that supplies the vertex indices for the replacement primitive.
+
+## Constraints
+
+A given primitive index **MUST NOT** appear in more than one primitive group.
+
+Each primitive in each group **MUST** use one of the following topology types, as specified by the `mode` property: 2 (line loop), 3 (line strip), 5 (triangle strip), or 6 (triangle fan). No other topology types are permitted.
+
+All primitives in a given group **MUST** have identical property values (e.g., attributes, material, mode, etc), with the exception of `indices`.
```

Comment:
Although it's technically implied here, it's still worth explicitly mentioning that if any primitive of a group has a `KHR_materials_variants` extension object, then all other primitives of the same group **MUST** have that extension with identical content.

Reply:
Clarified - thanks.